x-pack/plugins/security_solution/public/detections/components/host_isolation/unisolate.tsx

```diff
@@ -38,7 +38,11 @@ export const UnisolateHost = React.memo(
       return caseInfo.id;
     });
 
-    const { loading, unIsolateHost } = useHostUnisolation({ endpointId, comment, caseIds });
+    const { loading, unIsolateHost } = useHostUnisolation({
+      endpointId,
+      comment,
+      caseIds: caseIds.length > 0 ? caseIds : undefined,
```

Comment:
it looks like that `useHostUnisolation` is used only here, but in case it will be used elsewhere in the future, I'd push down this check into the inside of the hook, so it will be more robust against this bug

Reply:
@gergoabraham addressed here: https://github.com/elastic/kibana/pull/152644/commits/73b0ab02f772d70d81ae1f677e8092509ed92e44